spec/models/debate_spec.rb

```diff
@@ -48,28 +48,36 @@
 
   describe "#editable?" do
     let(:debate) { create(:debate) }
+    before(:each) { Setting.find_by(key: "max_votes_for_debate_edit").update(value: 3) }
 
     it "should be true if debate has no votes yet" do
       expect(debate.total_votes).to eq(0)
       expect(debate.editable?).to be true
     end
 
-    it "should be false if debate has votes" do
-      create(:vote, votable: debate)
-      expect(debate.total_votes).to eq(1)
+    it "should be true if proposal has less than limit votes" do
```

Comment:
Creo que aquí y en el siguiente debería decir "is true if *debate* has less than the limit votes"